src/ir/comp.rs

```diff
@@ -668,30 +690,92 @@ impl CompFields {
         );
     }
 
-    fn deanonymize_fields(&mut self) {
+    fn deanonymize_fields(&mut self, ctx: &BindgenContext, methods: &[Method]) {
```

Comment:
Maybe `assign_field_names` is a slightly better name for this method now? I'm not married to either name...